src/parser/parsers/souls.py

```diff
@@ -0,0 +1,78 @@
+from parser.maps import SOUL_UNLOCK_MAP
+
+
+class SoulUnlockParser:
```

Comment:
Brief description of what soul unlock means would be good

Reply:
Good point, added!